include/El/blas_like/level3.hpp

```diff
@@ -9,621 +9,25 @@
-
-// Hemm
-// ====
-template<typename T>
-void Hemm
-( LeftOrRight side, UpperOrLower uplo,
-  T alpha, const Matrix<T>& A, const Matrix<T>& B, T beta, Matrix<T>& C );
-
-template<typename T>
-void Hemm
-( LeftOrRight side, UpperOrLower uplo,
-  T alpha, const ElementalMatrix<T>& A, const ElementalMatrix<T>& B,
-  T beta,        ElementalMatrix<T>& C );
-
-// Herk
-// ====
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const Matrix<T>& A, Base<T> beta, Matrix<T>& C );
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const Matrix<T>& A, Matrix<T>& C );
-
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const ElementalMatrix<T>& A, 
-  Base<T> beta,        ElementalMatrix<T>& C );
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const ElementalMatrix<T>& A, ElementalMatrix<T>& C );
-
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const SparseMatrix<T>& A, 
-  Base<T> beta,        SparseMatrix<T>& C );
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const SparseMatrix<T>& A, 
-                       SparseMatrix<T>& C );
-
-template<typename T>
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const DistSparseMatrix<T>& A,
-  Base<T> beta,        DistSparseMatrix<T>& C );
-template<typename T> 
-void Herk
-( UpperOrLower uplo, Orientation orientation,
-  Base<T> alpha, const DistSparseMatrix<T>& A,
-                       DistSparseMatrix<T>& C );
-
-// Her2k
-// =====
-template<typename T>
-void Her2k
-( UpperOrLower uplo, Orientation orientation,
-  T alpha,       const Matrix<T>& A, const Matrix<T>& B, 
-  Base<T> beta,        Matrix<T>& C );
-
-template<typename T>
-void Her2k
-( UpperOrLower uplo, Orientation orientation,
-  T alpha, const Matrix<T>& A, const Matrix<T>& B, Matrix<T>& C );
-
-template<typename T>
-void Her2k
-( UpperOrLower uplo, Orientation orientation,
-  T alpha,      const ElementalMatrix<T>& A, const ElementalMatrix<T>& B,
-  Base<T> beta,       ElementalMatrix<T>& C );
-
-template<typename T>
-void Her2k
-( UpperOrLower uplo, Orientation orientation,
-  T alpha, const ElementalMatrix<T>& A, const ElementalMatrix<T>& B,
-                 ElementalMatrix<T>& C );
-
-// Multiply
-// ========
-// NOTE: The following routine multiplies a sparse matrix by a set of vectors
-//       and is obviously not a BLAS routine. However, it is a basic linear
-//       algebra routine making use of Elemental's core data structures, and
-//       so this is the natural placement
-template<typename T>
-void Multiply
-( Orientation orientation,
-  T alpha, const SparseMatrix<T>& A, const Matrix<T>& X,
-  T beta,                                  Matrix<T>& Y );
-
-template<typename T>
```

Comment:
It appears you have not replaced this prototype anywhere and that this is the problem.